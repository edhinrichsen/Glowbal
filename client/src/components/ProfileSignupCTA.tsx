import React from 'react';
import { Result } from 'antd';

import './ProfileSignupCTA.css';
import LoginButton from './LoginButton';

function ProfileSignupCTA() {
  return (
    <div className="profile-signup-cta">
      <Result
        status="info"
        title="Sign up or login to create a profile"
        extra={<LoginButton />}
      />
    </div>
  );
}

export default ProfileSignupCTA;
