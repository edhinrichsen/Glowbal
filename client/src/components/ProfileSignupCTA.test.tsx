import React from 'react';
import { shallow } from 'enzyme';
import ProfileSignupCTA from './ProfileSignupCTA';

describe('Testing ProfileSignupCTA', () => {
  it('renders without crashing', () => {
    const component = shallow(<ProfileSignupCTA />);
    expect(component).toMatchSnapshot();
    component.unmount();
  });
});
