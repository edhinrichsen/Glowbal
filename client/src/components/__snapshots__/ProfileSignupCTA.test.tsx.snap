// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing ProfileSignupCTA renders without crashing 1`] = `
<div
  className="profile-signup-cta"
>
  <Result
    extra={<LoginButton />}
    status="info"
    title="Sign up or login to create a profile"
  />
</div>
`;
